File: market-application/src/main/java/com/example/market/application/command/RecordSellerShippingCommand.java
package com.example.market.application.command;

import com.example.market.domain.shared.UserId;
import com.example.market.domain.trading.TradeId;

public record RecordSellerShippingCommand(
        UserId requestor,    // 본인 거래만 가능
        TradeId tradeId,
        String trackingNumber
) {}

File: market-application/src/main/kotlin/com/example/market/application/command/RecordSellerShippingCommand.kt
package com.example.market.application.command

import com.example.market.domain.shared.UserId
import com.example.market.domain.trading.TradeId

@JvmRecord
data class RecordSellerShippingCommand(
    val requestor: UserId,    // 본인 거래만 가능
    val tradeId: TradeId,
    val trackingNumber: String,
)
